Suggested edits (#39)
apeltzer/eager2-paper: content/03.results-and-discussion.md

## Scalability, Portability, and Efficiency

The re-implementation of EAGER into Nextflow offers a range of benefits over the
original custom pipeline framework.

Firstly, the new framework provides immediate integration of nf-core/eager into
various job schedulers in POSIX High-Performance-Cluster (HPC) environments,
cloud computing resources, as well as local workstations. This portability
allows users to run nf-core/eager regardless of the type of
computing insfrastructure or cluster size (if applicable), with minimal effort or configuration, which facilitates
reproducibility and therefore maintenance of standards within the field. This is
further assisted by the in-built compatibility with software environments and
containers such as Conda [@url:https://conda.io],

> Docker[@url:https://www.docker.com] and Singularity [@url:https://sylabs.io].

Reviewer: ```suggestion
Docker [@url:https://www.docker.com] and Singularity [@url:https://sylabs.io].
```

[@doi:10.1016/j.cub.2017.10.025]). When using a human reference genome,
nf-core/eager can now also can give estimates of the biological sex of a given
individual with Sex.DetEERRmine [@doi:10.1038/s41467-018-07483-5]. A dedicated
'endogenous DNA' calculator (endorS.py) is also included to provide an estimate 

> of the percentage of the total sequenced reads per library that map to the provided

Reviewer: I think this is less understandable, it was Wolfgang who re-wrote this as I thought we need to describe in very basic words what endogenous DNA means (as it's not a common term). @ZandraFagernas @MaxUlysse what do you think what do you think?

Author: Happy with either.

Reviewer: Lets go with the original. Can you revert?

reference sequence.

Given the large amount of sequencing often required to yield sufficient genome

> coverage from aDNA data, palaeogeneticists tend to use multiple (often differently

Reviewer: ```suggestion
coverage from aDNA data, palaeogeneticists tend to use multiple (differently
```

Not necessarily often, common but not often.

published under CC-BY licenses to allow for use in other training material
(example in {@fig:doc-image-example}). We hope this open-access resource will
make the study of aDNA more accessible to researchers new to the field, by
providing practical guidelines on how to evaluate the characteristics of aDNA 
and their effects on downstream analyses.

![Example of output interpretation documentation schematic images that can
assist new users in the interpretation to next-generation-sequencing aDNA
processing.](images/Fig2_fastqc_adapter_content.png){#fig:doc-image-example
width="70%"}

The development of nf-core/eager in Nextflow and the nf-core initiative will

> also allow for open-source development, while ensuring the high quality of community contributions to the pipeline. While Nextflow

Reviewer: EAGER1 was already open source? 
```suggestion
also improve open-source development, while ensuring the high quality of community contributions to the pipeline. While Nextflow
```

Author: It is still something that development in Nextflow and nf-core contribute to, even if it isn;t a change from eager 1, no?

Author: oh wait. your suggestion works
